core/nodejsActionBase/src/service.js

```diff
@@ -52,17 +57,27 @@ function NodeActionService(config) {
         return responseMessage(code, { error: errorMsg });
     }
 
+    /**
```

Comment:
is this used anywhere? the check for re-init is untouched below.

Reply:
dropped it